src/types/Types.tsx

```diff
@@ -200,3 +200,12 @@ export interface PageTree {
 export interface Membership {
   user: User;
 }
+
+export interface Group {
+  name: string;
+  slug: string;
+  description: string;
+  contact_email: string;
+  type: string;
+  leader: Partial<User>;
```

Comment:
Denne bør types med strict med akkurat hva som er i User, for eks:
```suggestion
  leader: Pick<User, 'user_id' | 'first_name' | 'last_name'>;
```

Hvis ikke vil man måtte sjekke hver gang man bruker user.* om fieldet er undefined eller ikke